src/firefly/html/firefly.html

```diff
@@ -12,33 +12,7 @@
     <script>
         window.firefly = {
             app: {
-                template: 'FireflyViewer',
-                menu: [
-                    {label:'Images', action:'ImageSelectDropDownCmd'},
-                    {label:'Catalogs', action:'IrsaCatalogDropDown'},
-                    {label:'Charts', action:'ChartSelectDropDownCmd'},
-                    {label:'Upload', action: 'FileUploadDropDownCmd'},
-                    //{label:'Workspace', action: 'WorkspaceDropDownCmd'}
-                    ],
-                options : {
-                    MenuItemKeys: {maskOverlay:true},
-                    catalogSpacialOp: 'polygonWhenPlotExist',
-                    charts: {multitrace: true, singleTraceUI: false},
-                    hips : {useForCoverage: true, useForImageSearch: true,
-                            hipsSources: 'irsa,cds',
-                            defHipsSources: {source: 'irsa', label: 'IRSA Featured'},
-                            mergedListPriority: 'Irsa'},
-                    workspace : {showOptions: false},
-                    imageMasterSourcesOrder: ['WISE', '2MASS', 'Spitzer'],
-                    coverage : { // example of using DSS and wise combination for coverage (not that anyone would want to combination)
-                        hipsSourceURL : 'http://alasky.u-strasbg.fr/DSS/DSSColor', // url
-                        imageSourceParams: {
-                            Service : 'WISE',
-                            SurveyKey: '1b',
-                            SurveyKeyBand: '4'
-                        }
-                    }
-                }
+                template: 'FireflyViewer'     // set to app mode, instead of api.
```

Comment:
Loi an I discuss moving all the defined options in firefly.html to firefly-dev.html.  This way firefly-dev.html can become an example of how to set options.  They will defined in firefly-dev.html even if they are not overriding the default.  Also we will add more and some documentation there as well.  This example will be very valuable of NED and new developers that are less familiar with options.